src/Logger/RedisCallInterceptor.php

```diff
@@ -87,4 +88,19 @@ private function flatten(array $arguments, array &$list): void
             }
         }
     }
+
+    /**
+     * Removes all non printable characters from input, returns default string if the whole input was non printable
+     *
+     * @param string $command The command to check
+     */
+    public function makePrintable(string $command): string
+    {
+        $sanitized = preg_replace('/[^[:print:]]/', '', $command);
+        if ($sanitized !== '') {
```

Comment:
This cannot happen, since part of $command is always method name, which is ASCII. Given this, I would suggest to inline preg_replace call directly.